src/features/search/state/searchActions.ts

```diff
@@ -24,8 +24,11 @@ function fetchSearchResults (searchParams: SearchParams): ApiAction {
   return {
     type: 'search',
     [CALL_API]: {
-      endpoint: 'search',
-      method: 'GET',
+      endpoint: 'registry/search',
+      method: 'POST',
+      body: {
+        q: searchParams.q,
```

Comment:
We add this to the body, yet retain the url query param too. The reason being, core operates on our RPC dialect and requires a `POST` and `JS/TS` requires a non nil body so I just put something there, empty object didn't pass for me. 

On the other hand internally core uses both `GET` and `POST` against cloud to get the search results, where cloud treats pretty much all requests as GET. Should clean up the core search code and remove the GET version and transition it all to POST only I guess.

Reply:
sounds great. Mind adding a note in a comment here?